### PR TITLE
Mage_Media - mini DOC block update

### PR DESCRIPTION
**Pull request by sreichel**
- doc blocks added/fixed
- PSR2 fixes (whitespaces, linebreaks, ...)

### CODE REVIEW

**Comment by sreichel** [comment]
Small one :)

File: app/code/core/Mage/Media/Model/File/Image.php [hunk 1]
```diff
@@ -111,7 +126,7 @@ public function getImage(Mage_Media_Model_Image $object)
      * Create tmp image resource for operations
      *
      * @param Mage_Media_Model_Image $object
-     * @return $this
+     * @return false|resource
```
Comment (kiatng) [inline app/code/core/Mage/Media/Model/File/Image.php:129]: Shouldn't it be `@return $this`?

File: app/code/core/Mage/Media/Model/Image.php [hunk 2]
```diff
@@ -72,7 +74,7 @@ protected function _construct()
      * Set media image config instance
      *
      * @param Mage_Media_Model_Image_Config_Interface $config
-     * @return unknown
+     * @return Mage_Media_Model_Image
```
Comment (kiatng) [inline app/code/core/Mage/Media/Model/Image.php:77]: Shouldn't it be `@return $this`?

File: app/code/core/Mage/Media/Model/File/Image.php (hunk 1 above)
Comment (sreichel) [reply app/code/core/Mage/Media/Model/File/Image.php:129]: Ne, it returns `        $resource = imagecreatetruecolor($object->getDestanationDimensions()->getWidth(), $object->getDestanationDimensions()->getHeight());
        return $resource;`

File: app/code/core/Mage/Media/Model/Image.php (hunk 2 above)
Comment (sreichel) [reply app/code/core/Mage/Media/Model/Image.php:77]: Thsi was right, but collin merged some seconds ago ... will be fixed in "Round 2".

File: app/code/core/Mage/Media/Model/File/Image.php (hunk 1 above)
Comment (kiatng) [reply app/code/core/Mage/Media/Model/File/Image.php:129]: Yes, indeed. Thanks!

File: app/code/core/Mage/Media/Model/Image.php (hunk 2 above)
Comment (kiatng) [reply app/code/core/Mage/Media/Model/Image.php:77]: Thanks for your effort.

**Pull request merged**